src/utils/dom.ts

```diff
@@ -32,7 +32,7 @@ export function getElementDir(el: HTMLElement): Direction {
 
 export function getElementProp(el: Element, prop: string, fallbackValue: any, crossShadowBoundary = false): any {
   const selector = `[${prop}]`;
-  const closest = crossShadowBoundary ? closestElementCrossShadowBoundary(selector, el) : el.closest(selector);
+  const closest = crossShadowBoundary ? closestElementCrossShadowBoundary(el, selector) : el.closest(selector);
```

Comment:
@jcfranco I'm not seeing crossShadowBoundary being used in any components. Maybe we can remove this here?

Just to note: `getElementProp` doesn't cross shadow boundary. This is probably ok since I see it used in child components only so they shouldn't cross the shadowDom.

However, I see a lot of use of `this.el.closest()` in things like checkbox and input so it seems like we should update those to use `closestElementCrossShadowBoundary` Do we want an issue to prioritize this?

Reply:
https://github.com/Esri/calcite-components/blob/19882020664c4354f3110367712cb496b5192253/src/components/calcite-input/calcite-input.tsx

https://github.com/Esri/calcite-components/blob/9e5c810532eec100fcb13e232a7949d61721bf11/src/components/calcite-checkbox/calcite-checkbox.tsx#L191

Reply:
https://github.com/Esri/calcite-components/blob/19882020664c4354f3110367712cb496b5192253/src/components/calcite-input/calcite-input.tsx#L263